### PR TITLE
WIP: Blur blocked people's profile pic

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/patchwork/issues/1077

So far only blurs the large image in the profile view, so there are a bunch of other cases to take care of. Should be pretty simple to repeat the pattern I've done so far, if someone else wants to finish this PR that'd be great.

### CODE REVIEW

**Comment by cinnamon-bun** [comment]
Another important case is this widget:

ssb-user-icon-block

[Notes here in #1077](https://github.com/ssbc/patchwork/issues/1077#issuecomment-500668998)

File: modules/page/html/render/profile.js [hunk 1]
```diff
@@ -142,7 +142,11 @@ exports.create = function (api) {
     ])
 
     var prepend = h('header', { className: 'ProfileHeader' }, [
-      h('div.image', api.about.html.image(id)),
+      h('div.image', {
+          classList: [when(contact.hidden, '-hidden-profile-image', null)]
```
Comment (mmckegg) [inline modules/page/html/render/profile.js]: Not a huge deal, but the (unwritten) convention for class names in mcss is to use camelCase e.g. `-hiddenProfileImage`

Multi-word classes aren't that common, but I found some examples here: [styles/base/profile-header.mcss](https://github.com/ssbc/patchwork/blob/master/styles/base/profile-header.mcss#L29)

**Comment by christianbundy** [comment]
@Powersource Is this still something you're working on?

**Comment by marylychee** [comment]
I just stumbled upon another potential case. 

Screen Shot 2019-11-28 at 9 33 17 pm

E.g. we should blur people who have been blocked by +5/10 of our friends??

**Comment by christianbundy** [comment]
I realized that since all of these PRs are outdated I've been ignoring the PR count and missed some new PRs. I'm going to close this for now, please re-open if you have the time / interest / energy to work on this again! :heart:

**Pull request closed without merging**